packs/equipment/gate-attenuator.json

```diff
@@ -6,7 +6,7 @@
+            "value": "<p><em>Gate attenuators</em> are typically worn near the body's core and are shaped like portals or passageways, making literal the elemental gates kineticists possess within their bodies. The appearance can vary from a simple disk with a hole in the middle to a design matching a city gate of a particular settlement. If you're a kineticist, the attenuator grants you a +1 item bonus to your impulse attack modifier (but not to your impulse DC). When you invest a<em> gate attenuator</em>, attune it to one element of your choice. Designs on the attenuator's surface transform to match that element, and the attenuator gains the element's trait until it's no longer invested or is attuned to a different element.</p>\n<hr />\n<p><strong>Activate - Elemental Spell <span class=\"pf2-icon\">2</span></strong> (concentrate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Effect</strong> The <em>gate attenuator</em> casts a 1st-rank spell, with a spell attack modifier of +7 and spell DC of 17. If you're a kineticist and the spell's element matches one of your kinetic elements, you can use your impulse attack modifier instead of the spell attack modifier or your impulse DC instead of the spell DC. The spell corresponds to the element the item is attuned to, and it gains that element's trait if it doesn't already have it:</p>\n<p><strong>air</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Gust of Wind]</em></p>\n<p><strong>earth</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Pummeling Rubble]</em></p>\n<p><strong>fire</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Dehydrate]</em></p>\n<p><strong>metal</strong> @UUID[Compendium.pf2e.spells-srd.Item.Thunderstrike]</p>\n<p><strong>water</strong><em>@UUID[Compendium.pf2e.spells-srd.Item.Snowball]</em></p>\n<p><strong>wood</strong> @UUID[Compendium.pf2e.spells-srd.Item.Flourishing Flora]</p>"
```

Comment:
```suggestion
            "value": "<p><em>Gate attenuators</em> are typically worn near the body's core and are shaped like portals or passageways, making literal the elemental gates kineticists possess within their bodies. The appearance can vary from a simple disk with a hole in the middle to a design matching a city gate of a particular settlement. If you're a kineticist, the attenuator grants you a +1 item bonus to your impulse attack modifier (but not to your impulse DC). When you invest a<em> gate attenuator</em>, attune it to one element of your choice. Designs on the attenuator's surface transform to match that element, and the attenuator gains the element's trait until it's no longer invested or is attuned to a different element.</p>\n<hr />\n<p><strong>Activate - Elemental Spell <span class=\"pf2-icon\">2</span></strong> (concentrate)</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Effect</strong> The <em>gate attenuator</em> casts a 1st-rank spell, with a spell attack modifier of +7 and spell DC of 17. If you're a kineticist and the spell's element matches one of your kinetic elements, you can use your impulse attack modifier instead of the spell attack modifier or your impulse DC instead of the spell DC. The spell corresponds to the element the item is attuned to, and it gains that element's trait if it doesn't already have it:</p>\n<p><strong>air</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Gust of Wind]</em></p>\n<p><strong>earth</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Pummeling Rubble]</em></p>\n<p><strong>fire</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Dehydrate]</em></p>\n<p><strong>metal</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Thunderstrike]</em></p>\n<p><strong>water</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Snowball]</em></p>\n<p><strong>wood</strong> <em>@UUID[Compendium.pf2e.spells-srd.Item.Flourishing Flora]</em></p>"
```

Reply:
Spell links italicized